usaspending_api/recipient/delta_models/recipient_lookup.py

```diff
@@ -26,7 +26,7 @@
 
 RECIPIENT_LOOKUP_DELTA_COLUMNS = {k: v["delta"] for k, v in RECIPIENT_LOOKUP_COLUMNS.items()}
 RPT_RECIPIENT_LOOKUP_DELTA_COLUMNS = {k: v["delta"] for k, v in RECIPIENT_LOOKUP_COLUMNS_WITHOUT_ID.items()}
-RECIPIENT_LOOKUP_POSTGRES_COLUMNS = {k: v["postgres"] for k, v in RECIPIENT_LOOKUP_COLUMNS.items()}
+RECIPIENT_LOOKUP_POSTGRES_COLUMNS = {k: v["postgres"] for k, v in RECIPIENT_LOOKUP_COLUMNS_WITHOUT_ID.items()}
```

Comment:
When did the postgres columns transition over to the column set that was _without ID_? I thought the idea was we were (maybe not permanently) keeping ID in there and letting it auto-populate with the sequence?

Reply:
The table is created via the `LIKE` command which created the `id` column that references the sequence. This statement is all of the columns we are looking to copy back from Delta to Postgres. Not having `id` specified means that it will be auto populated by the sequence.